python/pyspark/ml/regression.py

```diff
@@ -743,6 +743,18 @@ def treeWeights(self):
         """Return the weights for each tree"""
         return list(self._call_java("javaTreeWeights"))
 
```

Comment:
@holdenk does it make sense to add `trees`?

A quick look seems like this would work (substituting `DecisionTreeClassificationModel` in the case of `RandomForestClassificationModel`) - but I may be missing some complexity somewhere.
```python
def trees(self):
    return [DecisionTreeRegressionModel(m) for m in list(self._call_java("trees"))]
```